libsolidity/analysis/TypeChecker.cpp

```diff
@@ -724,7 +724,7 @@ bool TypeChecker::visit(InlineAssembly const& _inlineAssembly)
 				}
 				else if (requiresStorage)
 				{
-					m_errorReporter.typeError(6617_error, _identifier.location, "The suffixes _offset and _slot can only be used on non-constant storage variables.");
+					m_errorReporter.typeError(6617_error, _identifier.location, "The suffixes .offset and .slot can only be used on non-constant storage variables.");
```

Comment:
Is it still considered as suffixes? Maybe something like "The members "offset" and "slot" are only available for non-constant state variables."

Reply:
This is interesting point. Maybe we should change to members, as it sounds ore natural for access with  `.`